docs/apps/user_guide/tracks_explorer.md

```diff
@@ -1,32 +1,44 @@
+4. Click "Load data". An interactive map will appear of your dataset, where you can pan/zoom to explore the dataset. If a map does not load, view the status message. The message "Error.. Check options and try again" likely indicates that your filepath is incorrect.
+5. You can change the background map tile using the drop-down menu in the sidebar.
+6. Toggle the "Datashade tracks" checkbox in the sidebar to turn on/off the point density aggregation feature. When unchecked, individual semi-transparent points will be displayed. Note that for very large datasets, plotting will be slow if the "Datashade tracks" option is turned off.
+7. A boundary for the tracks dataset is shown on the map in red. The default option uses a rectangular boundary with a buffer size of 0.1 (scale relative to the extent of the tracks). You can adjust the buffer size or change the boundary shape to a convex hull using the widgets in the sidebar.
+8. To save the boundary to a .geojson file, scroll down and if needed, edit the file name and location under "Output file". Then simply click the "Save extent" button.
+
+## Requesting environmental data from NASA
+
+You can use a .geojson file from the Tracks Explorer app to submit a request for environmental data from NASA's Application for Extracting and Exploring Analysis Ready Samples (AρρEEARS) service. 
+1. Go to NASA AppEEARS: https://appeears.earthdatacloud.nasa.gov/ 
+2. These data are free, but you will need an EarthData account to request data. Register at https://urs.earthdata.nasa.gov/users/new, and then sign in with your EarthData account. 
+3. In the top menu bar, click "Extract > Area".
+4. Click "Start new request". The interface looks like this:
+
+![appeears_area_sample](../images/appeears_area_sample.png)
+
+5. In the Extract Area Sample page, first upload the .geojson file you created in the Tracks Explorer. Alternatively, you can draw a polygon on the page or upload another .geojson or .shp file. It will display a preview of the area for which data will be provided.
+6. Select a date range. You can choose calendar dates for the start and end of the period, or select "Is Date Recurring?" to select a specific range of dates across years. 
+7. Select the data layers you want. Type a product name (e.g., "MYD10A2") or description (e.g., "snow") and it will display a list summarizing relevant products. To review all options, available products are listed at https://appeears.earthdatacloud.nasa.gov/productsNASA data. Also see the searchable catalog of many of these products at https://lpdaac.usgs.gov/product_search/?status=Operational. Once you have selected a product(s) from the list, click the "+" button on each desired layer. The layers will appear in the "Selected Layers" box to the right.
```

Comment:
```suggestion
7. Select the data layers you want. Type a product name (e.g., "MYD10A2") or description (e.g., "snow") and it will display a list summarizing relevant products. To review all options, available products are listed at <https://appeears.earthdatacloud.nasa.gov/products>. Also see the searchable catalog of many of these products at <https://lpdaac.usgs.gov/product_search/?status=Operational>. Once you have selected a product(s) from the list, click the "+" button on each desired layer. The layers will appear in the "Selected Layers" box to the right.
```